pkg/kube/controllers/quarksjob/errand_reconciler.go

```diff
@@ -22,6 +22,8 @@ import (
 var _ reconcile.Reconciler = &ErrandReconciler{}
 
 const (
+	// MeltdownDuration is the duration of merging consecutive triggers.
+	MeltdownDuration = 10 * time.Second
```

Comment:
Can't we re-use `r.config.MeltdownDuration` and making that default to `10s` instead?

Reply:
No, right now, I am only changing the meltdownduration for two reconcilers. Other reconcilers still follow old meltdown implementation. Going step by step.

Reply:
I think we should have a name for that and not re-use MeltdownDuration, something like 'ConsecutiveTriggerMergeDuration' but better.
Also we picked this from BOSH, but it's now commonly associated with the Intel CPU vulnerability :(

Since we use 'meltdown' to describe a state where to many events are triggered and we then start to ignore events until it 'cools' down, this seems different. 
MeltdownPrevention, PreMeltdown, MeltdownBuffer, PeriodOfCalm..?

Reply:
Done.